pkgs/tools/backup/partclone/default.nix

```diff
@@ -4,13 +4,13 @@
 
 stdenv.mkDerivation rec {
   name = "partclone-${version}";
-  version = "0.2.89";
+  version = "0.3.11";
 
   src = fetchFromGitHub {
     owner = "Thomas-Tsai";
     repo = "partclone";
     rev = version;
-    sha256 = "0gw47pchqshhm00yf34qgxh6bh2jfryv0sm7ghwn77bv5gzwr481";
+    sha256 = "1gw47pchqshhm00yf34qgxh6bh2jfryv0sm7ghwn77bv5gzwr481";
```

Comment:
This sha256 is invalid, it should be `0bv15i0gxym4dv48rgaavh8p94waryn1l6viis6qh5zm9cd08skg`.